### PR TITLE
Basic logic

### PR DESCRIPTION
**Pull request by Jian-Min-Huang**
Simple Test Pass Version
Poor jQuery Frontend But Can Play

Next Version Will Prepare The Protocol

### CODE REVIEW

File: mafia-backend-mono/src/main/kotlin/org/common/marvel/mafia/config/RootConfig.kt [hunk 1]
```diff
@@ -1,86 +1,96 @@
...
-        val stringBuilder = StringBuilder()
-
-        clients.stream()
-                .map { v -> connectorManager.sessionIdAccountMap.get(v.sessionId.toString()) }
-                .filter { v -> v != null }
-                .forEach { v -> stringBuilder.append("<li>$v</li>") }
-
-        return stringBuilder.toString()
-    }
+//    private fun genUserList(clients: List<SocketIOClient>): String {
```
Comment (echoulen) [inline mafia-backend-mono/src/main/kotlin/org/common/marvel/mafia/config/RootConfig.kt]: remove

File: mafia-backend-mono/src/main/kotlin/org/common/marvel/mafia/config/RootConfig.kt [hunk 2]
```diff
@@ -1,86 +1,96 @@
...
-            connectorManager.sessionIdClientMap.put(it.sessionId.toString(), it)
-
-            it.sendEvent("advert_info", """[SYSTEM]: $clientIp Hello ! I'm server ! """)
-            server.allClients.stream()
-                    .forEach {
-                        it.sendEvent("users", genUserLis(server.allClients.toList()))
-                    }
+            connectorManager.onlineSession.add(it)
+            it.sendEvent(Cmd.System.name, """[SYSTEM]: Hello ! I'm server ! """)
+//            server.allClients.stream()
```
Comment (echoulen) [inline mafia-backend-mono/src/main/kotlin/org/common/marvel/mafia/config/RootConfig.kt]: remove

File: mafia-backend-mono/src/main/kotlin/org/common/marvel/mafia/config/RootConfig.kt [hunk 3]
```diff
@@ -1,86 +1,96 @@
...
-            val clientIp = it.remoteAddress
-            println("""$clientIp disconnected !""")
-
-            connectorManager.sessionIdClientMap.remove(it.sessionId.toString())
-            server.allClients.stream()
-                    .forEach {
-                        it.sendEvent("users", genUserLis(server.allClients.toList()))
-                    }
+            connectorManager.onlineSession.remove(it)
+//            server.allClients.stream()
```
Comment (echoulen) [inline mafia-backend-mono/src/main/kotlin/org/common/marvel/mafia/config/RootConfig.kt]: remove

File: mafia-backend-mono/src/main/kotlin/org/common/marvel/mafia/config/RootConfig.kt [hunk 4]
```diff
@@ -1,86 +1,96 @@
...
 
-            client.sendEvent("advert_info", """[SYSTEM]: ${data.account} login success ! """)
-            server.allClients.stream()
-                    .forEach {
-                        it.sendEvent("users", genUserLis(server.allClients.toList()))
-                    }
+            connectorManager.accountSessionMap[loginMsg.account] = client
+            connectorManager.sessionAccountMap[client] = loginMsg.account
+            client.sendEvent(Cmd.System.name, """[SYSTEM]: ${loginMsg.account} login success ! """)
+//            server.allClients.stream()
```
Comment (echoulen) [inline mafia-backend-mono/src/main/kotlin/org/common/marvel/mafia/config/RootConfig.kt]: remove

File: mafia-backend-mono/src/main/kotlin/org/common/marvel/mafia/service/ConnectorManager.kt [hunk 5]
```diff
@@ -1,15 +1,109 @@
...
+                choosenSessionQueue.stream().forEach {
+                    it.sendEvent(Cmd.Game.name, JsonUtils.writeValueAsString(GameProtocol(id, Type.StartGame.name, memberInfos)))
+                }
+            }
+        }
+    }
 
-    val accountSessionIdMap = HashMap<String, String>()
+    // not enough
+//    @Scheduled(cron = "*/10 * * * * *")
```
Comment (echoulen) [inline mafia-backend-mono/src/main/kotlin/org/common/marvel/mafia/service/ConnectorManager.kt]: remove?

**Comment by Jian-Min-Huang** [comment]
commit again

**Pull request merged**